images/linux/scripts/installers/java-tools.sh

```diff
@@ -38,9 +39,16 @@ if isUbuntu20 ; then
     apt-get update
 fi
 
-# Install only LTS versions.
-apt-get -y install adoptopenjdk-8-hotspot=\*
-apt-get -y install adoptopenjdk-11-hotspot=\*
+for JAVA_VERSION in "${JAVA_VERSIONS_LIST[@]}"
```

Comment:
Why `DEFAULT_JDK_VERSION=11` is still hardcoded above?
We should get rid of both conditions `isUbuntu20` and `isUbuntu16 || isUbuntu18`